listening-comp/backend/performance_tracker.py

```
@@ -0,0 +1,787 @@
+import sqlite3
+import os
+import json
+from datetime import datetime, timedelta
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)